packages/api/src/modules/predicateVersion/services.ts

```diff
@@ -106,11 +106,7 @@ export class PredicateVersionService implements IPredicateVersionService {
   }
 
   async findCurrentVersion(): Promise<PredicateVersion> {
-    return await PredicateVersion.findOne({
-      order: {
-        createdAt: 'DESC',
-      },
-    })
+    return await PredicateVersion.findOne({ where: { active: true } })
```

Comment:
Wouldn't it be necessary to sort the search by the version of the predicate in descending order, to ensure that it fetches the most recent active version?